packages/strapi-plugin-users-permissions/admin/src/components/Controller/index.js

```diff
@@ -14,9 +14,8 @@ import InputCheckbox from '../InputCheckboxPlugin';
 
 import { Header, Label, Separator, Wrapper } from './Components';
 
-function Controller({ actions, inputNamePath, isOpen, name }) {
+function Controller({ actions, inputNamePath, isOpen, name, inputSelected, setInputSelected }) {
```

Comment:
Can you remove the `{ useState }` import since it is not needed anymore?

Reply:
Bad idea to skip once the linter. ^^' 
I updated the branch and removed the import. :)

Reply:
Bad idea to skip once the linter. 
Updated the branch and removed the import. :) 

P.S. Sorry for the multiple force-pushes (if you got any notifications). I seem to have some auto formatting issues while commiting.